### PR TITLE
Update combo docs to include Blazor

### LINKED ISSUE #713: Move combo topics to their own folder (currently in grid) and make it Blazor-friendly

### PR DESCRIPTION
**Pull request by simeonoff**
Closes #713 

This PR must be tested against https://github.com/IgniteUI/igniteui-blazor-examples/pull/321

### CODE REVIEW

File: doc/en/components/inputs/combo/overview.md [hunk 1]
```diff
@@ -117,13 +142,55 @@ combo.select(['BG01', 'BG02', 'BG03', 'BG04']);
...
+    Placeholder="Pick a city" 
+    Data="Data" 
+    ValueKey="Id" 
+    DisplayKey="Name">
+</IgbCombo>
+
+@code {
+    private List<City> Data;
+    private IgbCombo Combo;
+    private object[] UKCities;
```
Comment (HUSSAR-mtrela) [inline doc/en/components/inputs/combo/overview.md:158]: change to "Cities"

File: doc/en/components/inputs/combo/overview.md [hunk 2]
```diff
@@ -117,13 +142,55 @@ combo.select(['BG01', 'BG02', 'BG03', 'BG04']);
...
+</IgbCombo>
+
+@code {
+    private List<City> Data;
+    private IgbCombo Combo;
+    private object[] UKCities; 
+
+    protected override void OnInitialized() {
+        this.Data = SampleData.Cities;
+        this.UKCities = new object[] { "UK01", "UK02", "UK03", "UK04", "UK05" };
```
Comment (HUSSAR-mtrela) [inline doc/en/components/inputs/combo/overview.md:162]: change to "Cities"

File: doc/en/components/inputs/combo/overview.md [hunk 3]
```diff
@@ -117,13 +142,55 @@ combo.select(['BG01', 'BG02', 'BG03', 'BG04']);
...
+    }
+
+    public void SelectUKCities() {
+        this.Combo.Select(UKCities);
```
Comment (HUSSAR-mtrela) [inline doc/en/components/inputs/combo/overview.md:166]: change to "Cities"

File: doc/en/components/inputs/combo/overview.md [hunk 4]
```diff
@@ -117,13 +142,55 @@ combo.select(['BG01', 'BG02', 'BG03', 'BG04']);
...
+    }
+
+    public void DeselectUKCities() {
+        this.Combo.Deselect(UKCities);
```
Comment (HUSSAR-mtrela) [inline doc/en/components/inputs/combo/overview.md:170]: change to "Cities"

**Comment by ChronosSF** [comment]
LGTM

**Review by ChronosSF — APPROVED** [review]

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**